content/04.results.md

```diff
@@ -0,0 +1,24 @@
+## Results
+
+### Comparing bioRxiv to PubMed Central
+
+#### Global View
+1. Provide table of top 100 terms that are different between bioRxiv and PubMed Central

```

Comment:
I wonder if a table is the best way to compare these. I think you might be able to come up with a graphical representation that is more communicative. There are a couple things to convey for each word, I think. Which repository it indicates. How frequent it is relative to other words. How specific it is to the repository in question. Some works might be very frequent and specific. Others might be less frequent and specific.